todo_app/data/ToDoItem.py

```diff
@@ -0,0 +1,49 @@
+_BASE_LIST_ID = '60a6c184fd2064526bdf740'
+_NOT_STARTED_LIST_ID = 'b'
+_BEING_DONE_LIST_ID = 'c'
+_COMPLETED_LIST_ID = 'd'
+
+class ToDoItem:
+
+    def __init__(self, item):
+        self.id = item['id']
+        self.listId = item['idList']
+        self.title = item['name']
+        self.transformStatus(item['idList'])
+
+    def transformStatus(self, idList=None):
+        """
+        Transforms the idList attribute of the REST API payload into a readable status and displayClass.
+        DisplayClass is used for determining the item class for html rendering.
+        
+        """
+        if not idList:
+            idList = self.listId 
```

Comment:
We could get rid of this & the argument, and just always use `self.listId` for simplicity - but this is also a really nice opportunity to use properties as we could define those to be calculated dynamically as they're requested - e.g. see M2's reading material - the @property decorator, or [this online guide](https://www.programiz.com/python-programming/property)

Reply:
Removed this and converted the function into a static method in commit 4f9b126